### PR TITLE
Fix minimized_round_trips in lookup runtime fields

### PR DESCRIPTION
**Pull request by dnhatn**
Today, we have disabled ccs_minimized_round_trips for lookup requests, under the assumption that cross-cluster lookups occur when ccs_minimized_round_trips is disabled in the main search request. However, this assumption does not hold true for cases where the search is local but the lookup happens remotely.

### CODE REVIEW

File: server/src/main/java/org/elasticsearch/action/search/FetchLookupFieldsPhase.java [hunk 1]
```diff
@@ -87,7 +87,7 @@ private void doRun(List<Cluster> clusters) {
                 : "lookup across clusters only if [ccs_minimize_roundtrips] is disabled";
             for (LookupField lookupField : cluster.lookupFields) {
                 final SearchRequest searchRequest = lookupField.toSearchRequest(clusterAlias);
-                searchRequest.setCcsMinimizeRoundtrips(false);
+                searchRequest.setCcsMinimizeRoundtrips(context.getRequest().isCcsMinimizeRoundtrips());
```
Comment (jimczi) [inline server/src/main/java/org/elasticsearch/action/search/FetchLookupFieldsPhase.java:90]: What about the assertion above?  Shouldn't it fail if `context.getRequest().isCcsMinimizeRoundtrips()` is true?

Comment (dnhatn) [reply server/src/main/java/org/elasticsearch/action/search/FetchLookupFieldsPhase.java:90]: That assertion should not fail as the cluster alias is empty.

Comment (jimczi) [reply server/src/main/java/org/elasticsearch/action/search/FetchLookupFieldsPhase.java:90]: Ah right sorry, I confused with the cluster alias that would be set on the lookup index name.

**Review by jimczi — APPROVED** [review]
LGTM

**Comment by dnhatn** [comment]
update branch

**Comment by dnhatn** [comment]
Thanks @jimczi

**Pull request merged**